genjava/models/jackson.go

```diff
@@ -67,15 +84,6 @@ func (g *JacksonGenerator) VersionModels(version *spec.Version, thePackage packa
 	return files
 }
 
-func jacksonImports(w *sources.Writer) {
```

Comment:
Also do not mix imports required for generated models classes with imports required for (de)serializing models using json libraries.